src/language-js/printer-estree.js

```diff
@@ -766,6 +766,8 @@ function printPathNoParens(path, options, print, args) {
         printTypeAnnotation(path, options, print)
       ]);
     }
+    case "V8IntrinsicIdentifier":
```

Comment:
Did you grep the file for “Identifier” to see if there are others to update?

Reply:
Yes.

Reply:
(others don't seem relevant, and, as turns out, even the JSON one is not)